package.json

```diff
@@ -37,6 +40,7 @@
     "gulp-tag-version": "^1.1.0",
     "gulp-uglify": "^1.0.1",
     "gulp-util": "^3.0.1",
+    "jsdoc": "https://github.com/jsdoc3/jsdoc/tarball/master",
```

Comment:
Using the tarball because official versions don't yet fully support ES2015. I have found this is stable enough for use though.